source/components/defining-a-component.md

```diff
@@ -36,7 +36,7 @@ Each component, under the hood, is backed by an element. By default
 Ember will use a `<div>` element to contain your component's template.
 To learn how to change the element Ember uses for your component, see
 [Customizing a Component's
-Element](../components/customizing-a-components-element).
+Element](../../components/customizing-a-components-element).
```

Comment:
This does not work (at least locally), but `../customizing-a-components-element` does work